lib/components/molecules/Quote/Quote.style.js

```diff
@@ -0,0 +1,22 @@
+import { css } from 'styled-components';
+
+const styles = css`
+  .quote {
+    &__mark {
+      font-size: 60px;
+      float: left;
+    }
+    &__content {
+      padding-top: 5px;
+      ${props => (props.contentStyle ? props.contentStyle : '')};
+    }
+
+    &__author {
```

Comment:
Lets use flex to control the layout

Reply:
But here in this component there is no layout to control. :D